class/defaults.yml

```diff
@@ -1,3 +1,11 @@
 parameters:
   nfs_client_provisioner:
-    namespace: syn-nfs-client-provisioner
+    namespace: syn-storage
```

Comment:
Not sure if this a good idea. I guess we somehow miss a concept for namespace naming.

Reply:
reverted to default naming, will need to be rolled out with care on clusters that already use the component